lib/presentation/ui/occurrences/occurrences_list_page.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:vost/common/event.dart';
+import 'package:vost/domain/models/occurrence_model.dart';
+import 'package:vost/presentation/ui/_base/base_page.dart';
+import 'package:vost/presentation/ui/occurrences/occurrences_item.dart';
+import 'package:vost/presentation/ui/occurrences/occurrences_list_bloc.dart';
+
+class OccurrencesList extends BasePage<OccurrencesListBloc> {
```

Comment:
The occurrences list should be displayed in the home page widget, so is it possible to move it there?

At the same time, do do you think it's best to put it all in the homepage bloc since it is all present in the same context?